EVENT.md

```diff
@@ -866,12 +866,8 @@ In the above example, the browser could have chosen to generate three reports:
 ## Storage limits
 
 The browser may apply storage limits in order to prevent excessive resource
-usage.
+usage. Details on the pending sources per origin limit and pending event-level reports per destination site limit can be found [here](https://github.com/WICG/attribution-reporting-api/blob/main/params/chromium-params.md).
```

Comment:
Instead of linking to the Chromium parameters 6 times in this document, could we just link to it once, e.g. in https://github.com/WICG/attribution-reporting-api/blob/main/EVENT.md#privacy-considerations ?

Reply:
I think it is redundant to link to it 6 times, but makes it a bit easier to read through since you can go directly to the parameters page while reading a specific section @apasel422

Reply:
Makes more sense to not link to the parameters doc multiple times and just in the privacy considerations section